components/Code/Code.tsx

```diff
@@ -1,22 +1,32 @@
+import { useConfig } from "nextra-theme-docs"
+import { useRouter } from 'next/router'
 import { Tabs } from "nextra/components";
 import React, { Children } from "react";
 
 interface ChildrenProps {
   children: React.ReactElement;
 }
 
-Code.Next = NextJsCode;
+Code.Next = NextCode;
```

Comment:
I think we could leave `Code.Next` as the app router one, and just have `Code.NextPages` in case some code examples need specific `pages` router docs 💭

Reply:
Yeah good idea

I think the implementation of these `Code` component(s) need cleaned up a bit more in general too thuogh (or maybe I just didn't undertand how you intended these to be used exactly haha).